dotnet/keyspaces-lambda-csharp/src/keyspaces-lambda-csharp/Function.cs

```diff
@@ -0,0 +1,39 @@
+using System;
+
+using Amazon.Lambda.Core;
+using Amazon.Lambda.SQSEvents;
+using keyspaceslambdacsharp;
+using Newtonsoft.Json;
+
+// Assembly attribute to enable the Lambda function's JSON input to be converted into a .NET class.
+[assembly: LambdaSerializer(typeof(Amazon.Lambda.Serialization.SystemTextJson.DefaultLambdaJsonSerializer))]
+
+namespace keyspaces_lambda_csharp
+{
+    /*
+     * Represents the Lambda Function Handler.
+     * Returns "200 OK" if connection with Keyspaces is established successfully.
+     * Returns "400 Bad Request" in case of an error/exception.
+     */
+    public class Function
```

Comment:
Call this something other than function... Lambda Sample, etc.  Function is a pretty overloaded term even with the namespace.

Reply:
Renamed to LambdaSample.